package/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "$schema": "https://json.schemastore.org/tsconfig",
+    "include": ["./src/**/*"],
+    "compilerOptions": {
+        "rootDir": "./src",
+        "module": "amd",
+        "moduleResolution": "node",
+        "outDir": "./out/amd",
+        "target": "ES2020",
+        "alwaysStrict": true,
+        "declaration": true,
+        "skipLibCheck": true,
+        "types": [
+            "monaco-editor-core/monaco",
+            "@kusto/language-service/Kusto.JavaScript.Client",
+            "@kusto/language-service-next/Kusto.Language.Bridge"
+        ]
```

Comment:
Removed triple slash directives in favor of this, as per the TypeScript docs recommendation: https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html#-reference-types-